x-pack/solutions/observability/plugins/synthetics/public/apps/synthetics/components/monitors_page/overview/overview/overview_grid.tsx

```diff
@@ -152,6 +153,12 @@ export const OverviewGrid = memo(
         {view === 'cardView' ? (
           <>
             <div style={groupField === 'none' ? { height: listHeight } : undefined}>
+              {loading && (
+                <>
+                  <EuiProgress size="xs" color="accent" />
+                  <EuiSpacer size="xs" />
+                </>
+              )}
```

Comment:
Why is this needed?

Reply:
I agree it doesn't seem to be needed; the view already shows a loading spinner when the user changes the space filter.

Reply:
this is needed when silent reload is happening.

Reply:
Do we want to show when silent reload is happening? For me there are too many loading indicators when it's not a silent reload.